src/serializers/JSONSerializer/index.ts

```diff
@@ -19,14 +19,14 @@ function getCircularReplacer() {
 
 // toStringers strigifies some specific types
 function toStringers(_: any, value: any) {
+    // non-null assertion - exit early
+    if (!value) return value
```

Comment:
this is a falsy check, you want (as your comment suggests) a null or undefined check
```suggestion
    // non-null assertion - exit early
    if (value === undefined || value === null) return value
```